pkg/security/apparmor/validate_test.go

```diff
@@ -72,9 +60,6 @@ func TestValidateBadHost(t *testing.T) {
 }
 
 func TestValidateValidHost(t *testing.T) {
```

Comment:
in this test still need a validator object. See build failures

Reply:
Ah, it seems I have removed the object by mistake. Fixed the error now.